README.md

```diff
@@ -54,6 +54,10 @@ To open up file scheme in URL parameter:
 
 * `SCREENIE_ALLOW_FILE_SCHEME`: true (default `false`).
 
+To delay the screenshot so that the browser can render the page complete.
```

Comment:
"Delay from the `load` event until the screenshot is taken. This can solve issues with rendering (i.e. rendering webfonts) not being complete before the screenshot."